apps/dc_tools/setup.py

```diff
@@ -25,6 +25,8 @@
         'datacube',
         'odc_index',
         'odc_io',
+        'odc_aws'
+        'odc_aio'
```

Comment:
is `odc_aio` legitimately needed? It pulls in `aiobotocore` which can be a pain.